contrib/ohi-release-notes/action.yml

```
@@ -11,8 +11,8 @@ inputs:
     description: Fail if the held toggle is active. Defaults to `true`.
     default: 'true'
   link-dependencies-dictionary:
-    description: Sets the link dependency dictionary.
-    default: ''
+    description: Sets the link dependency dictionary. By default common a dictionary is used.
```

IMO the `link-dependencies-dictionary` value can be a bit misleading.

We state that a common dictionary is used by default, but that is only true if `.github/rt-dictionary.yml` (the default value) does not exist in the repository, right? In other words, if no value is set for `link-dependencies-dictionary` but the `.github/rt-dictionary.yml` file does exist it will be used instead of the default being stored in the action.

I don't think there is a simpler way to keep it configurable at the same time we use the dictionary stored in the action by default so, maybe updating the field description is enough. WDYT?

```suggestion
    description: Sets the link dependency dictionary. `.github/rt-dictionary.yml` is used by default, if it does not exist a common dictionary is used.
```

---

yea that comment is more clear thanks! I'm not worried about this since I don't think is going to be used tbh, I would prefer not to add any complexity because of it for maintainability.